docker-compose.yml

```diff
@@ -113,7 +113,6 @@ services:
             resources:
                 limits:
                     cpus: '2.0'
-                    memory: 2G
```

Comment:
So if I understand, now we don't cap memory in compose at all - if you mention above that it can cause OOMs, is it safe to set no cap at all?